src/controllers/root.go

```diff
@@ -7,4 +7,5 @@ import (
 // RegisterControllers registers all controllers
 func RegisterControllers(router *mux.Router) {
 	NewProductController(router)
+	// TODO how come it's ok not to set this to something?
```

Comment:
It is basically just registering handlers w/ the router... nothing to do w/ the actual controller instance that is returned.